src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SalaryInfoTest {
+    private static final String[] rolesArray = {"Сергей", "Андрей", "София"};
```

Comment:
No need to specify type in variable name in java. Compiler helps us to know types. 

```suggestion
    private static final String[] roles = {"Сергей", "Андрей", "София"};
```

Reply:
done